src/antares/model/area.py

```diff
@@ -299,6 +299,9 @@ def get_reserves_matrix(self) -> pd.DataFrame:
 
     def get_misc_gen_matrix(self) -> pd.DataFrame:
         return self._area_service.get_misc_gen_matrix(self)
+    
+    def get_renewable_matrix(self, renewable_id: str) -> pd.DataFrame:
```

Comment:
I just realized this doesn't make sense here. As an user will do `renewable.get_timeseries()` and not area.get_ts(renewable_id) it doesn't make sense. So you can remove this. 
Also rename the method `get_timeseries()` as it would be clearer for the user

Reply:
Okay understood, i removed it and renamed get_renewable_matrxi from renewableCluster object